src/app/projects/components/project-settings/helper/label-helper.ts

```diff
@@ -75,8 +75,11 @@ export class LabelHelper {
 
         labelInput.value = '';
         labelInput.focus();
-        this.settings.requestTimeOut = true;
-        timer(100).subscribe(() => this.settings.requestTimeOut = false);
+        requestTimeOut = true;
```

Comment:
requestTimout is called by value so changes aren't set "sent back" to the calling function. Either use a wrapper object or return logic

Reply:
i think only by refernce can work because of the timer stuff later